relay/circuit-v2.md

```diff
@@ -254,6 +254,13 @@ connection for the duration of any reservations and tag it to prevent
 accidental termination according to its connection management policy.
 If a relay server becomes overloaded however, it may still drop a
 connection with reservations in order to maintain its resource quotas.
+  
+If more data than the limit specified in the `data` field is transferred
+over the relayed connection, or the relayed connection has been open for longer than `duration`, the relay should reset the stream to the source and the stream to the destination. If the reservation
+for the connection has expired the server may then apply any connection
```

Comment:
Who's the "server" here. Is it the relay? We probably shouldn't be using 2 different terms here.

The following sentence doesn't really make sense to me, as a relayed connection doesn't really show up as a normal connection in the connection manager.

Reply:
> as a relayed connection doesn't really show up as a normal connection in the connection manager.

That sounds a bit too implementation-specific to me? I do agree though that the sentence is superfluous. "Do what you'd normally do" isn't really worth "specifying" I'd say.